src/main/java/ladder/domain/Direction.java

```diff
@@ -0,0 +1,7 @@
+package ladder.domain;
+
+public enum Direction {
+  LEFT,
+  RIGHT,
+  STRAIGHT
+}
```

Comment:
Direction이 이동 방향으로의 정보를 가지고 있다면 Coordinates에서 right, left와 같은 각각의 함수가 필요 없지 않을까요? 

```suggestion
public enum Direction {
  LEFT(Coordinates.of(-1,0))
  
  ...
  
  public Coordinates move(Coordinates target) {
      return target.move(this.coordinates);
  }
}
```

Reply:
말씀해주신대로, Direction enum 객체에 추가하였습니다. :) 동시에 `Coordinates`가 가지고 있던 `downRight`과 같은 이동 로직들은 제거하였습니다. `right()` 메서드는 이동에서는 사용되지 않고 `Point` 의 `next` 팩터리 메서드에서 사용되어서 남겨두었습니다 :)